qutebrowser/browser/adblock.py

```diff
@@ -234,16 +234,13 @@ def _parse_line(self, line):
         parts = line.split()
         if len(parts) == 1:
             # "one host per line" format
-            host = parts[0]
-        elif len(parts) == 2:
-            # /etc/hosts format
-            host = parts[1]
+            hosts = [parts[0]]
         else:
-            log.misc.error("Failed to parse: {!r}".format(line))
-            return False
+            hosts = parts[1:]
 
-        if host not in self.WHITELISTED:
-            self._blocked_hosts.add(host)
+        for host in hosts:
+            if host not in self.WHITELISTED:
```

Comment:
This would now add things like `ip6-localhost` to the blocked hosts. I actually think the whitelisting should be changed so any domain not containing a period (plus those ending with `.localdomain`) is ignored. What do you think?

Reply:
That's a good idea, will add that